crates/swc_ecma_minifier/tests/projects/output/react-dom-17.0.2.js

```diff
@@ -8072,8 +8072,7 @@
         0 === workInProgressRootExitStatus && (workInProgressRootExitStatus = 5);
     }
     function commitRoot(root) {
-        var renderPriorityLevel = getCurrentPriorityLevel();
-        return runWithPriority$1(99, commitRootImpl.bind(null, root, renderPriorityLevel)), null;
+        return runWithPriority$1(99, commitRootImpl.bind(null, root, getCurrentPriorityLevel())), null;
```

Comment:
This isn't strictly safe, but probably close enough.  We can agree that the access of `runWithPriority$1` variable should be fine, but the `commitRootImpl.bind` might have a side-effect getter. I think that's ok?

Reply:
This is safe, because this is done by the sequential inliner.
We prove that `getCurrentPriorityLevel()` cannot modify `commitRootImpl.bind` by proving that `getCurrentPriorityLevel` does not have any direct/indirect reference to `commitRootImpl`